compiler/rustc_resolve/Cargo.toml

```diff
@@ -7,6 +7,8 @@ edition = "2021"
 
 [dependencies]
 bitflags = "1.2.1"
+once_cell = "1.10.0"
```

Comment:
Why use `once_cell` instead of `std::cell::OnceCell` or `std::sync::OnceLock`?

Reply:
This is an initial prototype for perf testing, I'm not even sure the code using this `once_cell` is still there.
Originally it was copypasted from rustdoc.